### PR TITLE
Fix message handling

### PR DESCRIPTION
**Pull request by rgiyf**
Bot was handling all message as chatterbot messages and spamming with DMs. The issue was with Eris alias [Message.guild](https://abal.moe/Eris/docs/Message). Changed it to **Message.channel.guild**.

### CODE REVIEW

**Comment by KingDGrizzle** [comment]
Whats interesting is, it doesn't happen to me. Anyways, we still needed to do that change one day

**Comment by rgiyf** [comment]
There was an error/warning that **msg.guild** is deprecated for some time now. Seems that this alias isn't working anymore. At least for some users. :)

**Comment by HilbertGilbertson** [comment]
@rgiyf Indeed. We had noticed this (see issue #31) as it was causing issues elsewhere. The alias has **not** been removed (see Eris source [here](https://github.com/abalabahaha/eris/blob/master/lib/structures/Message.js#L156); unless you are using the dev build of Eris) but in some instances where GAB has used it, the deprecation warning seems to interrupt the code and so errors result.

Looking over the PR, everything seems fine.  I cannot see any means by which it could cause any problems. I've scoured through every change (with particular focus on the messagecreate/delete/update files) and can confirm it is simply a mass find and replace job (no typos etc) that should only rectify existing issues. I'll leave a formal review in due course.

File: Commands/Public/countdown.js [hunk 1]
```diff
@@ -81,27 +81,27 @@ module.exports = (bot, db, config, winston, userDocument, serverDocument, channe
...
                         },
                         color: 0xFF0000,
-						description: `That countdown doesn't exist. Use \`${bot.getCommandPrefix(msg.guild, serverDocument)}${commandData.name} ${suffix}|<time>\` to create it.`
+						description: `That countdown doesn't exist. Use \`${bot.getCommandPrefix(msg.channel.guild, serverDocument)}${commandData.name} ${suffix}|<time>\` to create it.`
 					}
 				});
 			}
 		}
 	} else {
 		// const info = serverDocument.config.countdown_data.filter(countdownDocument => {
```
Comment (KingDGrizzle) [inline Commands/Public/countdown.js]: Could you remove this codeblock? I don't know why its still here

**Pull request merged**